app/config/default_layouts.yml

```diff
@@ -107,8 +107,15 @@ ezplatform_page_fieldtype:
             views:
                 blog:
```

Comment:
Do we still need this view?

Reply:
Yes because we are using this view in Places & Tastes landing page to display Blog posts